lib/modules/platform/azure/index.ts

```diff
@@ -406,6 +407,20 @@ export async function getBranchStatus(
   if (noOfPending) {
     return 'yellow';
   }
+  if (!internalChecksAsSuccess) {
+    if (
+      statuses.every(
+        (status: GitStatus) =>
```

Comment:
```suggestion
        (status) =>
```
do we really need the type here?

Reply:
Nope. There's quite a bit of this in the tests. I decided to keep it consistent for now